items/REINDRAKE_SC.json

```diff
@@ -35,22 +36,27 @@
       "drops": [
         {
           "id": "WHITE_GIFT",
-          "chance": "65%"
+          "chance": "100% + 65% per hit"
         },
         {
           "id": "GREEN_GIFT",
-          "chance": "25%"
+          "chance": "100% + 25% per hit"
         },
         {
           "id": "RED_GIFT",
-          "chance": "10%"
+          "chance": "10% + 10% per hit"
         },
         {
           "id": "PROSPERITY;1",
           "extra": [
             "Only drops for the player who spawned the Reindrake"
           ],
           "chance": "100%"
+        },
+        {
+          "id": "DYE_ICEBERG",
+          "extra": [],
```

Comment:
empty extra?

Reply:
![{85A5B0CB-C046-4882-9DBD-98290B7C32F4}](https://github.com/user-attachments/assets/339d7f2d-5d59-4a9f-947c-e0f99aa06b6f)

im really not sure why it's done like that either, i don't mind removing it but it's just copied and pasted from another mob